src/browser/BrowserSettingsWidget.cpp

```diff
@@ -123,7 +124,13 @@ void BrowserSettingsWidget::loadSettings()
     m_ui->supportKphFields->setChecked(settings->supportKphFields());
     m_ui->noMigrationPrompt->setChecked(settings->noMigrationPrompt());
     m_ui->useCustomProxy->setChecked(settings->useCustomProxy());
-    m_ui->customProxyLocation->setText(settings->replaceHomePath(settings->customProxyLocation()));
+    QString sPathToCustomProx = settings->customProxyLocation();
```

Comment:
Just use `customProxyPath` or `customProxyLocation` here?

Reply:
Removed the intermediate variable.
Code is equivalent to the code which was in the reviewed pull request.